koku/api/report/ocp/provider_map.py

```diff
@@ -1067,7 +1067,7 @@ def __init__(self, provider, report_type):
                             ),
                             "cost_units": Value("USD", output_field=CharField()),
                             "clusters": ArrayAgg(Coalesce("cluster_alias", "cluster_id"), distinct=True),
-                            "cost": Value(0, output_field=DecimalField()),
+                            # "cost": Value(0, output_field=DecimalField()),
```

Comment:
Do we want to remove this rather than comment it out?

Reply:
```suggestion
```